README.md

```diff
@@ -47,72 +49,58 @@ You can also initilze a client with a sub-merchant-id:
 
 ```ruby
 client = Bambora::Client.new(
-  api_key: ENV.fetch('BAMBORA_API_KEY'),
-  merchant_id: ENV.fetch('BAMBORA_MERCHANT_ID'),
+  ...
   sub_merchant_id: 'submerchantid',
 )
 ```
 
+The client is then passed to one of the `*Resource` classes, mentioned below, in order to make a network request.
+
 ### Profiles
 
 *Summary*: Payment profiles store confidential payment information. Transactions can be processed against profiles.
 *Bambora Docs*: <https://dev.na.bambora.com/docs/guides/payment_profiles/>
 
+To use the profiles API, you must create an instance of the `Bambora::V1::ProfileResource` class.
+
+```ruby
+client = Bambora::JSONClient.new(...)
+profiles = Bambora::V1::ProfileResource.new(client: client, sub_path: '/v1/profiles')
+```
+
+Once the resource instance has been instantiated, actions can be made against the API.
+
 #### Create a Profile
 
 ```ruby
-client.profile.create(
+profiles.create(
   {
-    'language': 'en',
-    'comments': 'hello',
-    'card':{
-      'name': 'Hup Podling',
-      'number': '4030000010001234',
-      'expiry_month': '12',
-      'expiry_year': '23',
-      'cvd': '123',
+    language: 'en',
+    comments: 'hello',
+    card: {
+      name: 'Hup Podling',
+      number: '4030000010001234',
+      expiry_month: '12',
+      expiry_year: '23',
+      cvd: '123',
     },
   },
 )
 # => {
-#       code: 1,
```

Comment:
Does the actual response differ from their docs, then? Or did I miss-read the docs?

Reply:
Which documentation were you reading from?

Reply:
@cassidycodes which documentation were you reading from?

Reply:
https://dev.na.bambora.com/docs/references/payment_APIs/v1-0-5/

You have to click around to get to the `response`.

![image](https://user-images.githubusercontent.com/1842877/67519120-2cd05200-f674-11e9-943d-436c8dce34b7.png)

Reply:
Right, I see. When I clicked through that I found under the `Profiles` accordion tab the response should look like this:

![image](https://user-images.githubusercontent.com/14965586/67519248-77ea6500-f674-11e9-9a06-c419a66c8a92.png)

Reply:
Oooh, right. I got my alliterative endpoint names confused.